spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -31,6 +31,8 @@
 @ConfigurationProperties(prefix = "spring.cloud.stream.kinesis.binder")
 public class KinesisBinderConfigurationProperties {
 
+	private String region = null;
```

Comment:
The `region` must be obtained from the `cloud.aws.region.static` property in the environment instead.
I mean not here in this class, but in the place you would like to obtain it.